tests/web/WebDriverTest.php

```diff
@@ -802,6 +802,16 @@ public function testSeeInFieldForTextarea()
         $this->module->amOnPage('/form/bug2921');
         $this->module->seeInField('foo', 'bar baz');
     }
+    
+    /**
+    * @Issue 4726
+    */
+    public function testClearField()
+    {
+        $this->module->fillField('#username', 'username');
```

Comment:
OUPS! You don't open a webpage before filling in a field
And I don't know which webpage you are going to open. 
That's why current tests on SemaphoreCI are failing